### ISSUE TITLE
can't find crate for `openssl_sys` on FreeBSD10.1

### ISSUE THREAD

**Issue by hnakamur** [dhuseby/cargo-bootstrap#3]
Hi @dhuseby
Thanks for sharing cargo-bootstrap script.

I tried it on FreeBSD 10.1 and got the following error.

```
cargo:registry:curl:curl-sys: /tmp/out/build_script_curl_sys-0_1_22
cargo:registry:curl:curl-sys: cargo:rustc-link-search=native=/usr/local/lib
cargo:registry:curl:curl-sys: cargo:rustc-link-lib=curl
cargo:registry:curl:curl-sys: 
...
cargo:registry:curl:curl-sys: 
cargo:registry:curl:curl-sys:  cmd: ['-L', 'native=/usr/local/lib', '-l', 'curl']
cargo:registry:curl:curl-sys:  env: {}
cargo:registry:curl:curl-sys: denv: {}

cargo:registry:curl:curl-sys:  env: {}
cargo:registry:curl:curl-sys: rustc /tmp/out/curl-sys-0.1.22/lib.rs --crate-name curl_sys --crate-type lib -C extra-filename=-0_1_22 --out-dir /tmp/out -L /tmp/out --extern libz_sys=/tmp/out/liblibz_sys-0_1_3.rlib --extern libc=/tmp/out/liblibc-0_1_8.rlib --extern pkg_config=/tmp/out/libpkg_config-0_3_4.rlib -l curl -L native=/usr/local/lib -l curl
/tmp/out/curl-sys-0.1.22/lib.rs:5:14: 5:39 error: can't find crate for `openssl_sys`
/tmp/out/curl-sys-0.1.22/lib.rs:5 #[cfg(unix)] extern crate openssl_sys;
                                               ^~~~~~~~~~~~~~~~~~~~~~~~~
error: aborting due to previous error
...

Exception:
 from ./bootstrap.py, line 895:
 build command failed: 101
```

Here is steps I tried:

```
git clone https://github.com/dhuseby/cargo-bootstrap /tmp/bootstrap
git clone https://github.com/rust-lang/crates.io-index /tmp/index
mkdir /tmp/out
git clone https://github.com/rust-lang/cargo /tmp/cargo
cp bootstrap.py /tmp/cargo
cd /tmp/cargo
./bootstrap.py --crate-index /tmp/index --target-dir /tmp/out --no-clean --no-clone --no-clean --target x86_64-unknown-freebsd
```

Could you tell me how to fix this error?
Thanks!

**Comment by dhuseby** [comment]
Let me give this a go on a local machine of mine and see if I can figure it out.

**Comment by hnakamur** [comment]
Thanks!
I'm using clang 3.4.1 and the latest rustc on github master branch.
https://github.com/rust-lang/cargo/issues/429#issuecomment-106070323

**Comment by dhuseby** [comment]
Fixed in: 8ae338c

Just grab the latest.  On FreeBSD 10.1, I was just able to build a cargo executable using this script and then build cargo from source using the exe as the --local-cargo.

**Comment by hnakamur** [comment]
Thanks!
